easybuild/easyconfigs/g/GenomeSTRiP/GenomeStrip-2.00.1747-foss-2016b.eb

```diff
@@ -0,0 +1,69 @@
+##
+## This is a contribution from Phoenix HPC Service, The University of Adelaide, Australia
+## Homepage: https://www.adelaide.edu.au/phoenix/
+##
+## Copyright::  Copyright 2014-2017 adelaide.edu.au/phoenix
+## Authors::    Robert Qiao <robert.qiao@adelaide.edu.au>, Exequiel Sepulveda <exequiel.sepulveda@gmail.com>
+## License::    Copyright © 2009 - 2015 Broad Institute, Inc. and Harvard Medical School. All rights reserved.
+##
+## Notes::
+###
+
+easyblock = 'Tarball'
+
+name = 'GenomeSTRiP'
+version = '2.00.1747'
+
+homepage = 'http://software.broadinstitute.org/software/genomestrip/'
+description = """Genome STRiP (Genome STRucture In Populations) is a suite of tools 
+for discovering and genotyping structural variations using sequencing data. 
+The methods are designed to detect shared variation using data from multiple individuals.
+"""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'usempi': True}
+
+sources = ['svtoolkit_2.00.1747.tar']
```

Comment:
@robqiao Please mention the location where you can download from in a comment above this `sources` line

Also, don't hardcode the version here, use `%(version)s`:

```python
sources = ['svtoolkit_%(version)s.tar']
```